x-pack/plugins/apm/public/components/shared/ManagedTable/index.tsx

```diff
@@ -60,7 +60,7 @@ function UnoptimizedManagedTable<T>(props: Props<T>) {
   const renderedItems = useMemo(() => {
     // TODO: Use _.orderBy once we upgrade to lodash 4+
     const sortedItems = sortItems
-      ? sortByOrder(items, sortField, sortDirection)
+      ? orderBy(items, sortField, sortDirection as any)
```

Comment:
I don't think you need `any` here if you pass the third arg as an array

```suggestion
      ? orderBy(items, sortField, [sortDirection])
```

![image](https://user-images.githubusercontent.com/209966/85831506-b2ba1500-b78e-11ea-9b91-f794db7fee67.png)